src/waveform/widgets/waveformwidgetabstract.h

```diff
@@ -43,4 +44,22 @@ class WaveformWidgetAbstract : public WaveformWidgetRenderer {
     friend class WaveformWidgetFactory;
 };
 
+/// GLWaveformWidgetAbstract is a WaveformWidgetAbstract & QGLWidget that has
+/// a GLWaveformRenderer member which requires initialization that must be
+/// deferred until Qt calls QGLWidget::initializeGL and cannot be done in the
+/// constructor.
+class GLWaveformWidgetAbstract : public WaveformWidgetAbstract, public QGLWidget {
+  public:
+    GLWaveformWidgetAbstract(const QString& group, QWidget* parent);
+
+  protected:
+    void initializeGL() override {
```

Comment:
Wtf this is getting called on Linux but not macOS....

Reply:
Did you tried a clean rebuild?